app/src/main/java/helper/Utility.java

```diff
@@ -0,0 +1,26 @@
+package helper;
+
+import android.app.Activity;
+import android.content.Context;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Toast;
+
+/**
+ * Created by Sukriti on 7/25/16.
+ */
+public class Utility {
```

Comment:
Good idea in encapsulating these common functions, but given how much they're doing, seems a little overkill.